sql/item_latents.sql

```diff
@@ -3457,6 +3457,7 @@ INSERT INTO `item_latents` VALUES (23685,399,18,52,5);   -- Pedagogy Loafers+3:
 INSERT INTO `item_latents` VALUES (23685,399,18,52,6);   -- Pedagogy Loafers+3: Weather: Enhances Celerity and Alacrity Effect +18% (THUNDER)
 INSERT INTO `item_latents` VALUES (23685,399,18,52,7);   -- Pedagogy Loafers+3: Weather: Enhances Celerity and Alacrity Effect +18% (LIGHT)
 INSERT INTO `item_latents` VALUES (23685,399,18,52,8);   -- Pedagogy Loafers+3: Weather: Enhances Celerity and Alacrity Effect +18% (DARK)
+INSERT INTO `item_latents` VALUES (23804,384,2500,30,0);   -- Poroggo Cass. +1
```

Comment:
Commenting only with the item name is not as good for review.
It would be better to comment what this is doing.
> Poroggo Cass. +1: WATERSDAY: HASTE_GEAR: 25%